frontend/src/types/ConfigType.tsx

```diff
@@ -19,7 +19,7 @@ enum ArgConfigType {
 const SupportedList: string[] = [
   // ArgConfigType.LLM_API_KEY,
   // ArgConfigType.LLM_BASE_URL,
-  ArgConfigType.WORKSPACE_DIR,
+  // ArgConfigType.WORKSPACE_DIR,
```

Comment:
so option to set workspace from the frontend is temporarily removed

Reply:
Correct--it's actually broken right now. We'll re-add it in a future PR